jekyll/_cci2/server/Installation/5-install-hardening-your-cluster.adoc

```diff
@@ -20,23 +20,39 @@ A server installation basically runs three different type of compute instances:
+As always, the rule is to make as many of the resources as private as possible, applies. If your users will access your CircleCI server installation via VPN, there is no need to assign any public IP addresses at all, as long as you have a working NAT gateway setup. Otherwise, you will need at least one public subnet for the `circleci-proxy` load balancer.
 
-NOTE: From server v4.3.0, we have replaced https://github.com/traefik/traefik-helm-chart[Traefik] with https://github.com/Kong/charts[Kong] as our reverse proxy. However, in order to minimize disruption when upgrading, we chose not to rename the service used by Kong. Although you will see a service named `circleci-server-traefik`, this service is actually for Kong.
+NOTE: From server v4.0.0, we have placed an nginx reverse proxy infront of https://github.com/Kong/charts[Kong] and exposed this as a kubernetes service names `circleci-proxy`. We did this to simplify our infrastructure by exposing one loadbalancer rather than 4. Nginx is now responsible routing the traffic to kong, vm-service, output-processor and nomad.
 
 However, in this case, it is also recommended to place Nomad clients and VMs in a public subnet to enable your users to SSH into jobs and scope access via networking rules.
 
 Currently, custom subnetting is not supported for GCP. Custom subnetting support will be available in a future update/release.
 
+NOTE: When using Amazon Certificate Manager (ACM), the name of nginx's service will be `circleci-proxy-acm` instead of `circleci-proxy`. If you have switched from some other method of handling your TLS certificates to using ACM, this change will recreate the loadbalancer and you will have to reroute your associated DNS records for your <domain> and app.<domain>.
+
 ## Network Traffic
 This section explains the minimum requirements for a server installation to work. Depending on your workloads, you might need to add additional rules to egress for Nomad clients and VMs. As nomenclature between cloud providers differs, you will probably need to implement these rules using firewall rules and/or security groups.
 
 Where you see "external," this usually means all external IPv4 addresses. Depending on your particular setup, you might be able to be more specific (for example, if you are using a proxy for all external traffic).
 
-It is assumed that you have configured the load balancers for Nomad, vm-service and output processor to be internal load balancers. This is the default.
-
 The rules explained here are assumed to be stateful and for TCP connections only, unless stated otherwise. If you are working with stateless rules, you need to create matching ingress or egress rules for the ones listed here.
 
+### Reverse Proxy Status
+You may wish to check the status of the services routing traffic in your CircleCI Server installation and alert if there are any issue. Since we use both nginx and kong in CircleCI Server, we expose the status pages of both via port 80
```

Comment:
```suggestion
### Reverse proxy status
You may wish to check the status of the services routing traffic in your CircleCI server installation and alert if there are any issues. Since we use both Nginx and Kong in CircleCI server, we expose the status pages of both via port 80.
```